MapleServer2/Packets/StorageInventory.cs

```diff
@@ -0,0 +1,139 @@
+            pWriter.WriteLong(item.Uid);
+            pWriter.WriteShort(slot);
+            pWriter.WriteInt();
+            pWriter.WriteItem(item);
+
+            return pWriter;
+        }
+
+        public static Packet Remove(long uid)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+            pWriter.WriteEnum(ItemStorageMode.Remove);
+            pWriter.WriteLong();
+            pWriter.WriteLong(uid);
+
+            return pWriter;
+        }
+
+        public static Packet Move(long srcUid, short srcSlot, long dstUid, short dstSlot)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+            pWriter.WriteEnum(ItemStorageMode.Move);
+            pWriter.WriteLong();
+            pWriter.WriteLong(srcUid);
+            pWriter.WriteShort(srcSlot);
+            pWriter.WriteLong(dstUid);
+            pWriter.WriteShort(dstSlot);
+
+            return pWriter;
+        }
+
+        public static Packet UpdateMesos(long amount)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+            pWriter.WriteEnum(ItemStorageMode.Mesos);
+            pWriter.WriteLong(amount);
+
+            return pWriter;
+        }
+
+        public static Packet LoadItems(Item[] items)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+
+            pWriter.WriteEnum(ItemStorageMode.LoadItems);
+            pWriter.WriteLong();
+            pWriter.WriteShort((short) items.Count(x => x != null));
+            for (short i = 0; i < items.Length; i++)
+            {
+                if (items[i] == null)
+                {
+                    continue;
+                }
+                pWriter.WriteInt(items[i].Id);
+                pWriter.WriteLong(items[i].Uid);
+                pWriter.WriteShort(i);
+                pWriter.WriteInt();
+                pWriter.WriteItem(items[i]);
+            }
+
+            return pWriter;
+        }
+
+        public static Packet ExpandAnim()
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+            pWriter.WriteEnum(ItemStorageMode.ExpandAnim);
+
+            return pWriter;
+        }
+
+        public static Packet Sort(Item[] items)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STORAGE_INVENTORY);
+            pWriter.WriteEnum(ItemStorageMode.Sort);
+            pWriter.WriteLong();
+            pWriter.WriteShort((short) items.Count(x => x != null));
+            for (short i = 0; i < items.Length; i++)
+            {
+                if (items[i] == null)
+                {
+                    continue;
+                }
+                pWriter.WriteInt(items[i].Id);
+                pWriter.WriteLong(items[i].Uid);
+                pWriter.WriteShort(i);
+                pWriter.WriteInt();
+                pWriter.WriteItem(items[i]);
```

Comment:
Seems like a duplicate of `LoadItems`, might it be useful to use a shared method?